src/trace_processor/rpc/httpd.cc

```diff
@@ -48,6 +48,7 @@ const char* kDefaultAllowedCORSOrigins[] = {
     "https://ui.perfetto.dev",
     "http://localhost:10000",
     "http://127.0.0.1:10000",
+    "file://",
```

Comment:
Maybe a comment here about how this is for desktop-style applications (Electron)?